src/Presentation/settings/settings.default.json

```diff
@@ -0,0 +1,6 @@
+{
+  "environment": "Development",
+  "Database": {
+    "ConnectionString": "server=localhost;port=3306;user=root;password=admin;database=hermes"
```

Comment:
Should we change connection string into: "Server=127.0.0.1,3306;Database=hermes;User=root;Password=admin;"